src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -987,6 +987,6 @@ def cnp_round(
         assert isinstance(lsbs_value, int)
 
         if lsbs_value > 0:
-            x = fhe.round_bit_pattern(x, lsbs_to_remove=lsbs_value)
+            x = fhe.round_bit_pattern(x, lsbs_to_remove=lsbs_value, exactness=fhe.Exactness.APPROXIMATE)
```

Comment:
what is this ? 🤔

Reply:
Yes not sure we want this to be the default. Looks like it adds another 20min to the tests.